MPF.Core/Modules/Redumper/Parameters.cs

```diff
@@ -857,17 +870,44 @@ public override List<string> GetLogFilePaths(string basePath)
                         logFiles.Add($"{basePath}.fulltoc");
                     if (File.Exists($"{basePath}.log"))
                         logFiles.Add($"{basePath}.log");
-                    if (File.Exists($"{basePath}.skeleton"))
-                        logFiles.Add($"{basePath}.skeleton");
                     if (File.Exists($"{basePath}.state"))
                         logFiles.Add($"{basePath}.state");
                     if (File.Exists($"{basePath}.subcode"))
                         logFiles.Add($"{basePath}.subcode");
                     if (File.Exists($"{basePath}.toc"))
                         logFiles.Add($"{basePath}.toc");
+
+                    // Include .hash and .skeleton for all files in cuesheet
+                    var cueSheet = new SabreTools.Serialization.Files.CueSheet().Deserialize($"{basePath}.cue");
+                    string? baseDir = Path.GetDirectoryName(basePath);
+                    if (cueSheet?.Files != null && baseDir != null)
+                    {
+                        foreach (CueFile? file in cueSheet.Files)
+                        {
+                            string? trackName = Path.GetFileNameWithoutExtension(file?.FileName);
+                            if (trackName != null)
+                            {
+                                string trackPath = Path.Combine(baseDir, trackName);
+                                if (File.Exists($"{trackPath}.hash"))
+                                    logFiles.Add($"{trackPath}.hash");
+                                if (File.Exists($"{trackPath}.skeleton"))
+                                    logFiles.Add($"{trackPath}.skeleton");
+                            }
```

Comment:
Please invert this `if`

```suggestion
                            string? trackName = Path.GetFileNameWithoutExtension(file?.FileName);
                            if (trackName == null)
                                continue;
                                
                            string trackPath = Path.Combine(baseDir, trackName);
                            if (File.Exists($"{trackPath}.hash"))
                                logFiles.Add($"{trackPath}.hash");
                            if (File.Exists($"{trackPath}.skeleton"))
                                logFiles.Add($"{trackPath}.skeleton");
```